src/ontology/cl-edit.owl

```diff
@@ -23188,14 +23193,15 @@ AnnotationAssertion(rdfs:comment obo:CL_0011106 "This type of cell is found in m
 AnnotationAssertion(rdfs:label obo:CL_0011106 "GABAnergic interplexiform cell")
 EquivalentClasses(obo:CL_0011106 ObjectIntersectionOf(obo:CL_0011104 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534)))
 
-# Class: obo:CL_0011107 (Muller cell)
+# Class: obo:CL_0011107 (obsolete Muller cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:NV"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:21911394"^^xsd:string) obo:IAO_0000115 obo:CL_0011107 "Astrocyte-like radial glial cell that extends vertically throughout the retina, with the nucleus are usually in the middle of the inner nuclear layer.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:NV"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:21911394"^^xsd:string) obo:IAO_0000115 obo:CL_0011107 "OBSOLETE: Astrocyte-like radial glial cell that extends vertically throughout the retina, with the nucleus are usually in the middle of the inner nuclear layer.")
+AnnotationAssertion(obo:IAO_0100001 obo:CL_0011107 "CL:0000636")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0011107 "Muller glia")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0011107 "Müller cell")
-AnnotationAssertion(rdfs:label obo:CL_0011107 "Muller cell")
-SubClassOf(obo:CL_0011107 obo:CL_0000681)
-SubClassOf(obo:CL_0011107 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000966))
+AnnotationAssertion(rdfs:comment obo:CL_0011107 "Term was duplicated, use CL:0000636 instead")
```

Comment:
For future, if there is a related github issue, its good practice to add a rdfs:seeAlso annotation to a GitHub issue as well.